app/Models/RockTheVoteReport.php

```diff
@@ -87,4 +105,19 @@ public function getPercentageAttribute()
     {
         return round(($this->current_index * 100) / $this->row_count);
     }
+
+    /**
+     * Creates a new report from since and before, and saves it to retry_report_id.
+     *
+     * @return RockTheVoteReport
+     */
+    public function createRetryReport()
+    {
+        $retryReport = self::createViaApi($this->since, $this->before);
+
+        $this->retry_report_id = $retryReport->id;
```

Comment:
If the retry also fails, is the idea that _it_ in turn would display it's own retry button which would lead to _it's_ retry ID -> thus creating a navigable chain?

Reply:
Yep! Not the most convenient way to navigate a failed job's journey, but we're looking for minimal footprint on this feature.